src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -2806,6 +2806,197 @@
 short-summary: List out all available versions of Bicep CLI.
 """
 
+helps['stack'] = """
+type: group
+short-summary: A deployment stack is a native Azure resource type that enables you to perform operations on a resource collection as an atomic unit. Deployment stacks are defined in ARM as the type Microsoft.Resources/deploymentStacks. (Version 1.9)
```

Comment:
The description of `short-summary` is too long. Could you please put some content into `long-summary`?